docs/tutorials/basic/symbol.md

```diff
@@ -71,20 +42,14 @@ To complete this tutorial, we need:
     ```
     pip install jupyter
     ```
-- GPUs - A section of this tutorial uses GPUs. If you don't have GPUs on your machine, simply
-set the variable gpu_device to mx.cpu().
+- GPUs (optional). A section of this tutorial uses GPUs. If you don't have GPUs on your machine, simply set the variable `gpu_device` to `mx.cpu()`.
```

Comment:
Should automatically change context depending on whether a GPU is available or not.